packages/base/lua/CellularSpace.lua

```diff
@@ -26,6 +26,114 @@ local terralib = getPackage("terralib")
 
 TeCoord.type_ = "Coord" -- We now use Coord only internally, but it is necessary to set its type.
 
+local function loadNeighborhoodGAL(self, data)
+	local file = openFile(data.source, "r")
+	if file == nil then resourceNotFoundError("file", file) end
+
+	local header = file:read()
+	local lineTest = CSVparseLine(header, "\t" )
+	local b
+	if lineTest[2] ~= nil then b = "\t"
+	else b = " " end
```

Comment:
you could implement the if above in a single line (36 and 37 together). If it is not in the same line, you should use the syntax

```lua
if then
    -- do
else
    -- else
end